internal/cli/log_streams.go

```diff
@@ -63,7 +63,7 @@ var (
 	httpContentFormat = Flag{
 		Name:         "HTTP Content Format",
 		LongForm:     "http-format",
-		Help:         "HTTP Content-Format header. Possible values: JSONLINES, JSONARRAY, JSONOBJECT.",
+		Help:         "HTTP Content-Format header. Possible values: jsonlines, jsonarray, jsonobject.",
```

Comment:
is this related with the domains CRUD?

Reply:
Not, but found out about that, and seemed too small a change to create a separate PR for, and having to find a reviewer. Ideally under different circumstances, I'd have made that into a separate PR. If you'd prefer it that way I can extract it.